tests/cpython-tests/Dockerfile

```diff
@@ -1,6 +1,4 @@
 FROM vtikoo/cpython:3.8-ubuntu as build
```

Comment:
Why don't we use the standard python:3.8-slim docker image?

If there is a need for a custom image, maybe for debugging purpose, please use the team's docker hub repo. https://hub.docker.com/orgs/mystikos

Reply:
> Why don't we use the standard python:3.8-slim docker image?

Running all the test suites requires a python build.

Reply:
We prefer building docker images from docker files over pulling them from external sources except when it takes a long time to build from the source. If we have to build from cpython source to get the test suite, let's do that. It doesn't take very long as I recall.

Even if we have to pull docker images from an external source, the preferred location is the team's docker hub repo, as mentioned above.

Reply:
Per offline discussion, instead of pulling an external prebuilt image, changed to building entirely from a Dockerfile.